extensions/amp-analytics/0.1/visibility-impl.js

```diff
@@ -159,27 +159,32 @@ export function getElement(ampdoc, selector, el, selectionMethod) {
     return null;
   }
 
+  const elWin = el.ownerDocument.defaultView;
   // Special case for root selector.
   if (selector == ':host' || selector == ':root') {
-    const elWin = ampdoc.win;
     const parentEl = elWin.frameElement && elWin.frameElement.parentElement;
```

Comment:
cc/ @avimehta here, I think he's the original author here, just to make sure we're not missing anything.